src/main/io/osd.c

```diff
@@ -1198,6 +1245,18 @@ static void osdDrawElements(void)
         osdDrawSingleElement(OSD_LOG_STATUS);
     }
 #endif
+
+#ifdef USE_OSD_STICK_OVERLAY
+    if (VISIBLE(osdConfig()->item_pos[OSD_STICK_OVERLAY_LEFT])) {
+        osdDrawStickOverlayAxisItem(OSD_STICK_OVERLAY_LEFT);
+        osdDrawStickOverlayCursor(OSD_STICK_OVERLAY_LEFT, THROTTLE, YAW);
```

Comment:
(this works only in mode 2)

Reply:
Now updated to support both mode 1 and mode 2.